sdk/communication/Azure.Communication.Identity/api/Azure.Communication.Identity.netstandard2.0.cs

```diff
@@ -25,6 +25,7 @@ public partial class CommunicationIdentityClientOptions : Azure.Core.ClientOptio
         public CommunicationIdentityClientOptions(Azure.Communication.Identity.CommunicationIdentityClientOptions.ServiceVersion version = Azure.Communication.Identity.CommunicationIdentityClientOptions.ServiceVersion.V2021_03_07) { }
         public enum ServiceVersion
         {
+            V2021_02_22_preview1 = 0,
```

Comment:
V2021_02_22_preview1 = 0, this version and enum value should be greater than the current GA version

Reply:
Resolved offline to leave this change as is (reason being this private preview version will be overridden by upcoming public preview version very soon, so not worth making the service version changes now)